api/provider/aws/test_aws/service_update_test.go

```diff
@@ -0,0 +1,92 @@
+package test_aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/golang/mock/gomock"
+	provider "github.com/quintilesims/layer0/api/provider/aws"
+	"github.com/quintilesims/layer0/api/tag"
+	awsc "github.com/quintilesims/layer0/common/aws"
+	"github.com/quintilesims/layer0/common/config/mock_config"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func TestServiceUpdate(t *testing.T) {
```

Comment:
Having `ServiceUpdate` and `ServiceScale` update is really confusing - we should always be consistent that `ServiceScale` only exists in the cli command package.

Reply:
✅